src/pages/saksbehandling/steg/sats/Sats.tsx

```diff
@@ -77,25 +77,30 @@ const utledSats = (values: FormData, harEPS: boolean, epsAlder?: Nullable<number
     return null;
 };
 
-const schema = yup.object<FormData>({
-    delerSøkerBolig: yup.boolean().defined().when('epsFnr', {
-        is: null,
-        then: yup.boolean().required(),
-    }),
-    mottarEktemakeEllerSamboerSU: yup
-        .boolean()
-        .defined()
-        .test('eps mottar SU', 'Feltet må fylles ut', function (mottarSu) {
-            const { epsFnr, epsAlder } = this.parent;
+const getValidationSchema = (eps: Nullable<Person>) => {
+    return yup.object<FormData>({
+        delerSøkerBolig: yup
+            .boolean()
+            .defined()
+            .test('deler søker bolig', 'Feltet må fylles ut', function (delerSøkerBolig) {
+                if (!eps) {
+                    return delerSøkerBolig !== null;
+                }
+                return true;
+            }),
+        mottarEktemakeEllerSamboerSU: yup
+            .boolean()
+            .defined()
+            .test('eps mottar SU', 'Feltet må fylles ut', function (mottarSu) {
+                if (eps && eps.alder && eps.alder < 67) {
```

Comment:
Er det ikke enklare att bare mosa ihop det med optional chaining?
```suggestion
                if (eps?.alder < 67) {
```